pkg/policies/dataplane/resources/fluxmeter/flux-meter.go

```diff
@@ -138,52 +139,26 @@ func (fluxMeterFactory *fluxMeterFactory) newFluxMeterOptions(
 }
 
 func (fluxMeter *FluxMeter) setup(lc fx.Lifecycle, prometheusRegistry *prometheus.Registry) {
-	metricLabels := make(map[string]string)
-	metricLabels[metrics.FluxMeterNameLabel] = fluxMeter.GetFluxMeterName()
-
 	lc.Append(fx.Hook{
 		OnStart: func(_ context.Context) error {
-			// Initialize a prometheus histogram metric
-			fluxMeter.histMetricVec = prometheus.NewHistogramVec(prometheus.HistogramOpts{
-				Name:        metrics.FluxMeterMetricName,
-				Buckets:     fluxMeter.buckets,
-				ConstLabels: prometheus.Labels{metrics.FluxMeterNameLabel: fluxMeter.fluxMeterName},
-			}, []string{
-				metrics.DecisionTypeLabel,
-				metrics.StatusCodeLabel,
-				metrics.FeatureStatusLabel,
-			})
-			// Register metric with Prometheus
-			err := prometheusRegistry.Register(fluxMeter.histMetricVec)
-			if err != nil {
-				log.Error().Err(err).Msgf("Failed to register metric %+v with Prometheus registry", fluxMeter.histMetricVec)
-				return err
-			}
-
```

Comment:
@DariaKunoichi,
See how a new FluxMeter metrics Vector is created for each FluxMeter mini Fx app. Need to restore the same semantics in order to support different histogram buckets per FluxMeter